tasks/phpstan/phpstan.neon

```diff
@@ -0,0 +1,11 @@
+parameters:
+	tmpDir: temp/phpstan
+	bootstrap: tasks/phpstan/bootstrap.php
+	ignoreErrors:
+		- '#Access to an undefined property#'
```

Comment:
Why did you decide to ignore this error?

Reply:
Unfortunately, this is a widespread habit in the current code. It should be definitely fixed in the code itself (maybe next hacking day) but for the time being it can help the purpose of getting the checks green. Maybe there should be a comment about that?

Reply:
Yeah, a comment would be great. Can you add it, please?